gamestonk_terminal/common/technical_analysis/volume_view.py

```diff
@@ -140,42 +140,54 @@ def plot_adosc(
     else:
         bar_width = timedelta(minutes=int(s_interval.split("m")[0]))
 
+    divisor = 1000000
```

Comment:
Not something to be done now.

Just for future, Python3 has this feature that allows you to add "_" in between to make a number more readable.

E.g. `1_000_000`